cmd/stack_lint.go

```diff
@@ -25,14 +25,12 @@ import (
 func newStackLintCmd(rootConfig *RootCommandConfig) *cobra.Command {
 	var lintCmd = &cobra.Command{
 		Use:   "lint",
-		Short: "Lint your stack.",
-		Long: `Verifies your stack conforms with the structure of a valid Appsody stack. 
+		Short: "Check your stack structure.",
+		Long: `Check that the stack structure of your stack is valid. Error messages indicate critical issues in your stack structure, such as missing files, directories, or stack variables. Warning messages suggest optional stack enhancements.
```

Comment:
Too many "stacks" I think :)  Suggest
"Check that the structure of your stack is valid."  (unless there are other kinds of structure!)